flaskr/templates/base.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
When you can, its general practice to include screenshots of the functionality you're adding.

Less common in infra work, but common in product work.

Reply:
![image](https://user-images.githubusercontent.com/43485746/85329439-636e9e80-b487-11ea-914a-d8469e1b8065.png)
![image](https://user-images.githubusercontent.com/43485746/85329478-75e8d800-b487-11ea-80c4-c95cf6b5aede.png)